### PR TITLE
Pagerduty urgencies

### PR DESCRIPTION
**Pull request by jtsaito**
Adding support for Pagerduty resource's `incident_urgency_rule` for `type` `constant`.

This is passing the acceptance tests.

~~However, still checking if having a resource's attribute as optional and with default simultaneously is ok by Terraform convention.~~ (The combination of defaults and optional values in resources' schema definitions appears to be common .)

(Based on https://github.com/hashicorp/terraform/pull/10766)

### CODE REVIEW

**Comment by jtsaito** [comment]
@jansiwy Please check this so that I can test adding the urgencies to the Pagerduty service for bookshelf.

File: builtin/providers/pagerduty/resource_pagerduty_service_integration_test.go [hunk 1]
```diff
@@ -117,6 +117,11 @@ resource "pagerduty_service" "foo" {
   auto_resolve_timeout    = 1800
   acknowledgement_timeout = 1800
   escalation_policy       = "${pagerduty_escalation_policy.foo.id}"
+
+	incident_urgency_rule {
+		type = "constant"
+		urgency = "high"
+	}
```
Comment (jansiwy) [inline builtin/providers/pagerduty/resource_pagerduty_service_integration_test.go]: please unify indentation

File: builtin/providers/pagerduty/resource_pagerduty_service_integration_test.go [hunk 2]
```diff
@@ -162,6 +167,11 @@ resource "pagerduty_service" "foo" {
   auto_resolve_timeout    = 3600
   acknowledgement_timeout = 3600
   escalation_policy       = "${pagerduty_escalation_policy.foo.id}"
+
+  incident_urgency_rule { 
+			type    = "constant"
+			urgency = "high"
```
Comment (jansiwy) [inline builtin/providers/pagerduty/resource_pagerduty_service_integration_test.go]: please unify indentation

**Comment by jtsaito** [comment]
Note: one acceptance test (`TestAccPagerDutySchedule_Multi`) is failing but it also fails on master and on the original master at hashicorp.

**Pull request merged**